app/src/main/java/cloudcity/CloudCityOMNTApplication.java

```diff
@@ -2,11 +2,26 @@
 
 import android.app.Application;
 
+import cloudcity.dataholders.MetricsPOJO;
+
 public class CloudCityOMNTApplication extends Application {
+    public static final String TAG = CloudCityOMNTApplication.class.getSimpleName();
+
+    private Iperf3Monitor iperf3Monitor;
 
     @Override
     public void onCreate() {
         super.onCreate();
         CloudCityParamsRepository.initialize(getApplicationContext());
+
+        Iperf3Monitor.initialize(getApplicationContext());
+        iperf3Monitor = Iperf3Monitor.getInstance();
```

Comment:
nope. what are we defending from, the Iperf3Monitor doesn't throw anything. this suggestion is terrible and doesn't differ much from "you should enclose every block of code in a try/catch because, who knows, something bad might happen... resolving.